Master-OB-OpenAPI.json

```
@@ -14446,6 +14420,521 @@
+            "x-ob-sample-value": {}
+          }
+        ]
+      },
+      "ContractES": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/Contract"
+          },
+          {
+            "type": "object",
+            "description": "Contract describing the energy services between the service provider and the service requestor.",
+            "properties": {
+              "EffectiveDate": {
+                "$ref": "#/components/schemas/EffectiveDate"
+              },
+              "ExpirationDate": {
+                "$ref": "#/components/schemas/ExpirationDate"
+              },
+              "Aggregator": {
+                "$ref": "#/components/schemas/Aggregator"
+              },
+              "ContractTerms": {
+                "$ref": "#/components/schemas/ContractTerms"
+              },
+              "ProgramName": {
+                "$ref": "#/components/schemas/ProgramName"
+              },
+              "SignUpAmount": {
+                "$ref": "#/components/schemas/SignUpAmount"
+              }
+            }
+          }
+        ]
+      },
+      "PaymentRateCapacity": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "Capacity payment per unit of power",
+            "x-ob-item-type": "MonetaryPerPowerItemType",
+            "x-ob-item-type-group": "",
+            "x-ob-usage-tips": "Use this to store the performance based incentive in $/kW",
+            "x-ob-sample-value": {}
+          }
+        ]
+      },
+      "PaymentRateEnergy": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementNumber"
+          },
+          {
+            "type": "object",
+            "description": "Payment rate for the energy transferred",
+            "x-ob-item-type": "MonetaryPerEnergyItemType",
+            "x-ob-item-type-group": "",
+            "x-ob-usage-tips": "",
+            "x-ob-sample-value": {}
+          }
+        ]
+      },
+      "PaymentRateDuration": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementNumber"
+          },
+          {
+            "type": "object",
+            "description": "Payment rate per unit of time. For e.g. 200 USD/hour",
+            "x-ob-item-type": "MonetaryPerDurationItemType",
+            "x-ob-item-type-group": "",
+            "x-ob-usage-tips": "",
+            "x-ob-sample-value": {
+              "Unit": "USD_per_h",
+              "Value": 200
+            }
+          }
+        ]
+      },
+      "PaymentRatePerEvent": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementNumber"
+          },
+          {
+            "type": "object",
+            "description": "Payment rate for every event",
```

```suggestion
            "description": "Payment for every event",
```

Per-event payment, correct? Or is this something like $/hr during the event?